Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp

```diff
@@ -527,6 +527,12 @@ namespace EditorPythonBindings
         return GetWrappedObjectRepr();
     }
 
+    pybind11::ssize_t PythonProxyObject::GetWrappedObjectHash()
```

Comment:
With python 3.10 + pybind, any object that has an unordered collection does not provide a default __hash__ implementation, thus producing a 'not hashable' error for the whole object. This object maintains a collection unordered methods and properties that causes this error. Since the methods arent important to the instance that this class represents, this function is a manual implementation of the hash calculation that ignores these 2 underordered collections.